applications/callflow/src/callflow.hrl

```diff
@@ -6,6 +6,13 @@
 -include_lib("kazoo_call/include/kapps_call_command_types.hrl").
 -include_lib("kazoo_documents/include/kazoo_documents.hrl").
 
+-record(cf_bl_action, {action :: kz_term:api_binary()
```

Comment:
I don't think `bl` is sufficient; I would prefer to see `blacklist` to immediately remind the reader what is being written about here.

Reply:
I renamed record name. But modules functions still have name `bl_action`.

Reply:
Please update any `bl` to `blacklist`; no need for terseness here.

Reply:
Renamed except `kzd_users:bl_strategy/2`. Reason already exist `kzd_users:blacklist_strategy/2` generated by `make apis` and imported from template.
Optionally i can not import autoenerated `kzd_users:blacklist_strategy/2`.